_sources/GeneralIntro/intro-TheWayoftheProgram.rst

```diff
@@ -12,33 +12,20 @@ Introduction: The Way of the Program
 ====================================
 
 The goal of this book is to teach you to understand and create computer programs in Python.
-You will use that ability to shed light on how people and computers are alike
-and different as processors of information. This book provides
-a human-centric view of computer programming.
 
 Understanding computer programs requires algorithmic, mechanistic thinking.
 Programs specify mechanistic sequences of actions to perform; when executed,
 they transform input data into output data. They execute very reliably, and 
-very fast, but not creatively. Computers do what you tell them to do, not you
-mean for them to do. Understanding computer code, and especially debugging it,
-involves a lot of mental simulation of what will actually happen, not what you 
-wish would happen. This way of thinking combines some of the best features of 
-mathematics, engineering, and natural science. Like mathematicians, programmers
-use formal languages to denote ideas (specifically computations). Like engineers, 
-they design things, assembling components into systems and evaluating tradeoffs 
-among alternatives. Like scientists, they observe the behavior of complex systems, 
-form hypotheses, and test predictions.
+very fast, but not creatively. Computers do what you tell them to do, not what you
+mean for them to do. Thus, understanding computer code
+involves a lot of mental simulation of what will actually happen, not what you
+wish would happen.
  
 Writing computer programs requires not only mechanistic thinking but creative
 problem solving. Problem solving means the ability to formulate problems, think
 creatively about solutions, and express a solution clearly and accurately. As
 it turns out, the process of learning to program is an excellent opportunity to
-practice problem solving skills. That's why this chapter is called, *The Way of
-the Program*.
-
-On one level, you will be learning to program, a useful skill by itself. On
-another level, you will use programming as a means to an end. As we go along,
-that end will become clearer.
```

Comment:
Hmm, my team at google just lifted this text you've deleted as part of the intro script.  

I think we need to talk about the audience for this as I have a feeling you and I may not have the same audience in mind and thats going to be problematic.